components/Buttons/examples/ButtonsTypicalUse.m

```diff
@@ -30,9 +30,11 @@ - (MDCButton *)buildCustomStrokedButton {
   [button setBackgroundColor:[UIColor clearColor] forState:UIControlStateNormal];
   [button setTitleColor:[UIColor colorWithWhite:0.1f alpha:1] forState:UIControlStateNormal];
   button.inkColor = [UIColor colorWithWhite:0 alpha:0.06f];
-  button.layer.borderWidth = 1;
-  button.layer.borderColor = [UIColor blackColor].CGColor;
   button.disabledAlpha = 0.38f;
+
+  [button setBorderWidth:1.0 forState:UIControlStateNormal];
+  [button setBorderColor:[UIColor blackColor] forState:UIControlStateNormal];
```

Comment:
Should stroke color be the same color as Title Color?
[UIColor colorWithWhite:0.1f alpha:1]